src/utils/sortObjectKeys.test.ts

```diff
@@ -1,10 +1,17 @@
-// sortObjectKeys.test.ts
+import { describe, expect, it, jest } from "@jest/globals";
 import { sortObjectKeys } from "./sortObjectKeys";
 import { SortOrder } from "../types/settings";
-import { beforeEach, describe, expect, it, jest } from "@jest/globals";
+
+jest.mock("../types/settings", () => ({
+  SortOrder: {
+    ASC: "A-Z",
+    DESC: "Z-A",
+    NONE: "none",
+  },
+}));
```

Comment:
move this to jest.setup.js so it will be mocked on all the tests

Reply:
you can put it here: https://github.com/heliomar-pena/i18n-populator/blob/60d9cf5c92cabc84ca11977a576a6e6b2b572f0d/jest.setup.ts#L30-L36